modules/misc/src/materials/ArrheniusMaterialProperty.C

```diff
@@ -0,0 +1,104 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ArrheniusMaterialProperty.h"
+
+#include "PhysicalConstants.h"
+
+#include "libmesh/utility.h"
+
+registerMooseObject("MiscApp", ArrheniusMaterialProperty);
+registerMooseObject("MiscApp", ADArrheniusMaterialProperty);
+
+template <bool is_ad>
+InputParameters
+ArrheniusMaterialPropertyTempl<is_ad>::validParams()
+{
+  InputParameters params = Material::validParams();
+
+  params.addClassDescription(
+      "Arbitrary material property of the sum of an arbitary number (i) of "
+      "Arrhenius functions A_i * exp(-Q_i / R / T), where A_i is the frequency "
+      "factor, Q_i is the activation energy, and R is the gas constant.");
```

Comment:
```suggestion
      "Arbitrary material property of the sum of an arbitary number ($i$) of "
      "Arrhenius functions $A_i * \\exp{-Q_i / (RT)}$, where $A_i$ is the frequency "
      "factor, $Q_i$ is the activation energy, and $R$ is the gas constant.");
```

This will improve the MooseDocs render of this description.